i18nextICU.js

```diff
@@ -2066,7 +2066,10 @@ function getDefaults() {
     memoize: true,
     memoizeFallback: false,
     bindI18n: '',
-    bindI18nStore: ''
+    bindI18nStore: '',
+    parseErrorHandler: function parseErrorHandler(err, key, options) {
+      return typeof options.defaultValue !== 'undefined' ? options.defaultValue : key;
```

Comment:
Actually, maybe instead of the default value should it return the unformatted string?

Reply:
Good point. At least it'll be in the right language even if it has the un-replaced variables in there.